src/gt4py/next/iterator/embedded.py

```diff
@@ -1260,7 +1263,7 @@ def _get_axeses(field):
         return tuple(itertools.chain(*tuple(_get_axeses(f) for f in field)))
     else:
         assert is_located_field(field)
-        return (field.axes,)
+        return (_get_axes(field),)
```

Comment:
Why not? The field can not be a tuple anymore anyway.
```suggestion
        return (field.__gt_dims__,)
```

Reply:
Since this is the only place were I directly access __gt_dims_ (all other go through `_get_axes`) I thought it makes sense, but probably doesn't make sense. I'll revert.